float64[] right_shoulder
string[] right_shoulder_joints
float64[] left_shoulder
string[] left_shoulder_joints
float64[] right_elbow
string[] right_elbow_joints
float64[] left_elbow
string[] left_elbow_joints